packages/saltcorn-cli/src/commands/create-user.js

```diff
@@ -5,7 +5,11 @@
 const { Command, flags } = require("@oclif/command");
 const { cli } = require("cli-ux");
 const { maybe_as_tenant } = require("../common");
+const { loadAllPlugins } = require("@saltcorn/server/load_plugins");
```

Comment:
Imports that import the database module (or does so transitively e.g. server) and therefore initialise the database connection should be done inside the methods. I can't remember exactly what it is that goes wrong if you don't do this but one of the commands requires this. having the imports outside the class declaration in just one cli command means that the database will be connected in all of the commands before running. 

This is also why the functions in common do their imports inside the functions.

However you can make this much shorter by using init_some_tenants. this was only recently introduced and has not been implemented in all the commands but have a look at configuration-check as an example